content/docs/command-reference/pipeline/index.md

```diff
@@ -1,47 +0,0 @@
-# pipeline
-
-A set of commands to manage
-[pipelines](/doc/tutorials/get-started/data-pipelines):
-[show](/doc/command-reference/pipeline/show) and
-[list](/doc/command-reference/pipeline/list).
-
-## Synopsis
-
-```usage
-usage: dvc pipeline [-h] [-q | -v] {show,list} ...
-
-positional arguments:
-  COMMAND
-    show         Show pipeline.
-    list         List pipelines.
-```
-
-## Description
-
-A data pipeline, in general, is a series of data processing
-[stages](/doc/command-reference/run) (for example console commands that take an
-input and produce an <abbr>output</abbr>). A pipeline may produce intermediate
-data, and has a final result. Machine learning (ML) pipelines typically start a
-with large raw datasets, include intermediate featurization and training stages,
-and produce a final model, as well as accuracy
-[metrics](/doc/command-reference/metrics).
-
-In DVC, pipeline stages and commands, their data I/O, interdependencies, and
-results (intermediate or final) are specified with `dvc add` and `dvc run`,
-among other commands. This allows DVC to restore one or more pipelines of stages
-interconnected by their dependencies and outputs later. (See `dvc repro`.)
-
-> DVC builds a dependency graph
-> ([DAG](https://en.wikipedia.org/wiki/Directed_acyclic_graph)) to do this.
```

Comment:
Actually some of this info should be preserved in the `dvc dag` cmd ref for now because it's used as the place to explain what pipelines are: many other docs link to here around the word "pipeline". In any case some of this info should be preserved somewhere, I think. For now putting it in `dvc dag` seems appropriate to me.

Reply:
@jorgeorpinel Added.